sample/src/main/java/io/fotoapparat/sample/ActivityJava.java

```diff
@@ -169,27 +171,26 @@ private void switchCameraOnClick(View view) {
         view.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
+                activeCameraBack = !activeCameraBack;
                 fotoapparat.switchTo(
-                        front(),
+                        getOtherCamera(),
```

Comment:
`getCameraPosition` is more explicit since the "other" part is done 2 lines above. However a simple `activeCameraBack ? back() : front()` would be a very good idea too here